properties/math.py

```diff
@@ -132,7 +132,8 @@ def validate(self, instance, value):
                 'Array validation is only implmented for wrappers that are '
                 'subclasses of numpy.ndarray'
             )
-        if value.dtype.kind not in (TYPE_MAPPINGS[typ] for typ in self.dtype):
+        allowed_kinds = ''.join(TYPE_MAPPINGS[typ] for typ in self.dtype)
```

Comment:
Okay, so this should be safe b/c the setter on `.dtype` pre-validates that the specified dtypes are in the `TYPE_MAPPINGS` anyway, right?